### ISSUE TITLE
Using new Kraken PR Pi Image with Kerberos doesnt work.

### ISSUE THREAD

**Issue by O-I812** [krakenrf/krakensdr_docs#8]
Hello. I do have a new Kraken SDR, and it is wonderful. However, I have a custom built Passive RADAR setup using an original Kerberos SDR. I am using the latest RPi 4 image on a RPi4 connected to the Kerberos SDR. I have SSH'd into the RPi4 with the new image and updated the start.sh to boot in passive radar mode. After boot, I use the web page to load the "kerberos_default" DAQ file, however when I give it 10 minutes to configure, the DAQ Subsystem Status still shows red and there is no activity. I have also tried to start and stop processing, and rebooted the RPi4, but nothing seems to update and it doesn't work. I have checked the Wiki and troubleshooting section, however there is no description for this issue. My understanding is the new software will work with the Kerberos SDR, please let me know if I am not doing something correctly.

**Comment by krakenrf** [comment]
Hi, in passive radar mode you should be using the same prefixed "pr_" preconfig files that Kraken uses. They should work on both devices.

The kerberos_default preconfig file is only for DoA.

For Kerberos you might need to also set "Calibration Track Mode" to "No Tracking" in the DAQ settings, but I think it's possible that you will not need to do that. If you experience the software getting stuck on calibration about about 10-20 minutes, set that option to "no tracking" and try again.

**Comment by O-I812** [comment]
Hi, thank you for the help! Unfortunately, I still cannot get it to work. I loaded the default "pr_2ch_2pow21" file, and then I restarted the DAQ chain, and the GUI stated it was "Restarting...", but after 10 minutes nothing changed. I attempted to refresh the web browser, but then it was blank, I waited for a few more minutes, but nothing changed (after refreshing). Next, I rebooted the RPi 4, and the GUI came back, however when I "start processing" to connect to the SDR's, nothing happens like before. I also loaded file "pr_2ch_2pow20" and I was able to refresh the screen, but after about 10 minutes, nothing changed and the "Connection Status:" still states "Restarting...". Just before this was was using the original Kerberos SDR Debian image and the passive radar worked just fine, so I am confident that the hardware works and is functional. Is there any data I can help collect for you to try and troubleshoot? Using ssh, I poll the SDR's (rtl_test) and I see the 4 Rafael SDR's, so I believe they are connected. Any help is appreciated!

**Comment by krakenrf** [comment]
Did you flash the Kerberos with the EEPROM information required for the new software?

Info here: https://github.com/krakenrf/krakensdr_doa, under heading "KerberosSDR Setup (KrakenSDR users Ignore)".

Just realized this isn't on the Wiki, I'll add it now.

**Comment by krakenrf** [comment]
If you've done that already and you get the same issues, can you please take a look in the krakensdr_pr/ui.log file, and the heimdall_daq_fw/Firmware/_logs files and paste any information here?

**Comment by O-I812** [comment]
Ah, okay, thank you, I have not flashed the EEPROM with the new code, I will give that a try. I assume when I do this, the old Kerberos software will not be compatible? I am using 4 external RF coax switches that are driven with a RPi relay Hat for the noise source calibration, and I turn them off and on via a python script on the OS desktop. I assume that I will need to use the SSH command line to execute the python scripts from now on if I use the new Kraken OS image (as there is no desktop)? I will let you know how the EEPROM goes and report back the logs if I am not able to get it working.

**Comment by krakenrf** [comment]
The old Kerberos code will still be compatible even after modifying the EEPROM. The old code doesn't look for serial numbers, it just goes by enumeration. But on the KrakenSDR software we're using serials.

If you're using the Kraken Pi 4 image with the third party Kerberos switches it should automatically switch them in and out whenever the noise source turns on/off.

**Comment by O-I812** [comment]
Hello. I followed the Wiki as you suggested, and it worked! I flashed the eeprom like instructed and now the Kraken software works with the Kerberos as far as I can tell. The polling section seems to be updating and the FFT screen shows the spectrum and updates. On the passive RADAR screen it seems to always be red, no matter what I do (I even selected the auto setup, but no change), however I am sure that is operator error, so I will experiment more until I figure it out. Thank you for your time and help, I will close this issue, and thank you for updating the Wiki with the information!
